clang/unittests/AST/CommentLexer.cpp

```diff
@@ -1453,6 +1453,129 @@ TEST_F(CommentLexerTest, HTML19) {
   ASSERT_EQ(tok::newline,      Toks[2].getKind());
 }
 
+TEST_F(CommentLexerTest, HTML20) {
```

Comment:
can we add tests for 
```<
a
>
```
To make sure we don;t do something unreasonable - even though it's not a valid tag?